content/js/copy-to-clipboard.js

```diff
@@ -1,94 +1,86 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) Jenkins contributors
+
```

Comment:
The script is heavily based on 
https://gitlab.com/antora/antora-ui-default/-/blame/master/src/js/06-copy-to-clipboard.js
which is under MPL 2.0.

Since  this PR is for refactoring, let's not add any license header.